hikari/impl/entity_factory.py

```diff
@@ -4711,3 +4712,39 @@ def deserialize_auto_mod_rule(self, payload: data_binding.JSONObject) -> auto_mo
             exempt_channel_ids=[snowflakes.Snowflake(id_) for id_ in payload["exempt_channels"]],
             exempt_role_ids=[snowflakes.Snowflake(id_) for id_ in payload["exempt_roles"]],
         )
+
+    #####################
+    # SOUNDBOARD MODELS #
+    #####################
+
+    @typing_extensions.override
+    def deserialize_soundboard_sound(self, payload: data_binding.JSONObject) -> soundboard_models.SoundboardSound:
+        emoji: emoji_models.CustomEmoji | emoji_models.UnicodeEmoji | None = None
+        emoji_id = payload.get("emoji_id")
+        if emoji_id:
+            emoji_id = snowflakes.Snowflake(emoji_id)
+        emoji_name = payload.get("emoji_name")
+        if emoji_id and emoji_name:
+            emoji = emoji_models.CustomEmoji(
+                id=emoji_id,
+                name=emoji_name,
+                is_animated=False,  # FIXME: I am unsure if this is correct, but I have no clue on how I could validate this. # noqa: TD001, E501
+            )
+        if emoji_id and not emoji_name:
+            emoji = emoji_models.CustomEmoji(
+                id=emoji_id,
+                name="",  # FIXME: I am unsure if this is correct, but I have no clue on how I could validate this. # noqa: TD001, E501
+                is_animated=False,  # FIXME: I am unsure if this is correct, but I have no clue on how I could validate this. # noqa: TD001, E501
+            )
+        if not emoji_id and emoji_name:
+            emoji = emoji_models.UnicodeEmoji(emoji_name)
```

Comment:
Given we are not given `animated`, then lets always set it as False (Resolves the FIXMEs on it)

There will never be the case of `emoji_id and not emoji_name`, so that branch can be removed